src/unwrap.jl

```diff
@@ -300,56 +306,26 @@ end
 function get_border_range(size_img::NTuple{N, T}, border_dim, border_idx) where {N, T}
     border_range = [2:(size_img[dim]-1) for dim=1:N]
     border_range[border_dim] = border_idx:border_idx
-    return tuple(border_range...)
+    return Tuple(border_range)
 end
 
-function calculate_pixel_reliability(pixel_image::AbstractArray{Pixel{T}, N}, pixel_index, pixel_shifts, range) where {T, N}
-    sum_val = zero(T)
-    for pixel_shift in pixel_shifts
-        @inbounds sum_val += wrap_val(pixel_image[pixel_index+pixel_shift].val - pixel_image[pixel_index].val, range)^2
-    end
+function calculate_pixel_reliability(pixel_image::AbstractArray{Pixel{T},N}, pixel_index, pixel_shifts, range) where {T,N}
+    @inline rel_contrib(shift) =
+        let pix_val = pixel_image[pixel_index].val, rg = range, img = pixel_image, ind = pixel_index
+            @inbounds wrap_val(img[ind+shift].val - pix_val, rg)^2
+        end
```

Comment:
```suggestion
    pix_val = pixel_image[pixel_index].val
    @inline rel_contrib(shift) =
            @inbounds wrap_val(pixel_image[pixel_index+shift].val - pix_val, range)^2
```
Also, have you checked whether the `@inline`  is needed?